src/internal/group_api/requests.rs

```diff
@@ -11,13 +12,12 @@ use crate::internal::{
     IronOxideErr, RequestAuth, RequestErrorCode, SchnorrSignature,
 };
 use chrono::{DateTime, Utc};
+use internal::auth_v2::AuthV2Builder;
```

Comment:
Seems like this might be able to merged with the above import? (I know it wasn't your fault).